pages/index.tsx

```diff
@@ -0,0 +1,537 @@
+import { FunctionComponent } from 'react';
+
+import PageLink, { CallToAction } from '../components/PageLink';
+
+import AnalyticsManagerIcon from '../images/svg/analyticsmanager.svg';
+import DevIcon from '../images/svg/dev.svg';
+
+import StickyBox from 'react-sticky-box';
+
+import styles from './index.module.scss';
+
+function emptyRowElement() {
+  return <div style={{ display: 'flex', padding: '15px' }} />;
+}
+
+const LandingPage: FunctionComponent = () => {
+  const tocContent = (
+    <div>
+      {' '}
+      <p>
+        <a href="#stage1" className={styles.tocItem}>
+          1. Tracking plan management – no engineering required
+        </a>
+      </p>
+      <p>
+        <a href="#stage2" className={styles.tocItem}>
+          2. Tracking plan management with validation – one time SDK install
+          required
+        </a>
+      </p>
+      <p>
+        <a href="#stage3" className={styles.tocItem}>
+          3. End-to-end tracking plan collaboration and implementation workflows
+        </a>
+      </p>
+      <p>
+        <a href="#workspace" className={styles.tocItem}>
+          Explore your Avo workspace
+        </a>
+      </p>
+      <p>
+        <a href="#guides" className={styles.tocItem}>
+          Comprehensive data design and implementation guides
+        </a>
+      </p>
+    </div>
+  );
+
+  return (
+    <div className={styles.grid}>
+      <div>
+        <h1 className={styles.label}>Getting started with Avo</h1>
+        <p className={styles.text}>
+          The Avo workflows are layered. The 3 steps to adopt Avo can be adopted
+          as a whole or in part – depending on your use case. Each layer of the
+          workflow provides value and each of them requires different amount of
+          engineering contribution.
+        </p>
+
+        <p className={styles.text}>
+          For example, if you already have a framework for analytics
+          implementation and implementation data bugs are not a problem for you,
+          you can stay at the second adoption step.
+        </p>
+
+        <div className={styles.tocInline}>{tocContent}</div>
+
+        <h2 id="stage1" className={styles.label}>
+          1. Tracking plan management – no engineering required
+        </h2>
+        <p className={styles.text}>
+          Use Avo instead of your spreadsheet to manage and discuss your
+          tracking plan changes. Avo is designed for analytics, so it&apos;s way
+          better than the spreadsheet for managing your tracking plan.
+        </p>
+
+        <p className={styles.text}>
+          This adoption step does not require any developer input, so you can
+          adopt it in the data design team without getting a developer buy-in.
+        </p>
+        <h3 className={styles.label}>Creating tracking plan in Avo</h3>
```

Comment:
I feel like this one is slightly too big, it's competing too much with with the step 1. header

Reply:
Same applies to all the following sub-headers below